recipes/thermohubclient/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: thermohubclient
+  version: {{ version }}
+
+source:
+#   url: https://github.com/gdmiron/thermohubclient/archive/develop.tar.gz 
+#   sha256: 3519a2333f7a91f8f9dff2440a84208393a92119bca8de1ac1b71c50ac580b26
+  url: https://github.com/thermohub/thermohubclient/archive/v{{ version }}.tar.gz
+  sha256: 8a6901e34fcd87e88c098134750807f8cbf95080ce45f078bf2ff48955cf5f15
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja  # [win]
+    - make  # [unix and osx]
```

the ``unix`` selector will be true for ``osx`` or ``linux``
just ``unix`` here should be enough :)
```suggestion
    - make  # [unix]
```